src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java

```diff
@@ -360,10 +370,14 @@ public Object onTraversalSuccess() {
         }
 
         if (refCreator != null) {
-            try {
-                refCreator.writeMissingIntervals(intervalArgumentGenomeLocSortedSet);
-            } catch (IOException ioe) {
-                throw new GATKException("Error writing missing intervals", ioe);
+            if ((gqStatesToIgnore.size() != 1) || (!gqStatesToIgnore.contains(GQStateEnum.ZERO))) {
```

Comment:
Question about the logic. @RoriCremer - related to the discussion above about `dropAboveGqThreshold`.

If someone wants to drop more than one GQ state and one of them is ZERO, should it also drop the missings?

Reply:
the only use case for dropAboveGqThreshold that I know of, was for testing or for older reblocking versions. When we used data that had been re-blocked using reblocking version 1(?) -- and therefore had GQ0-60

No one should be dropping more than one GQ state unless the data has GQ50 and GQ60 in it and they are trying to do something with the reblocking version that expects a max of GQ40

Reply:
I think there are a lot of params in here that were used for testing, or for use cases that dont care about or that we now fully circumvent by something in our WDLs etc

And the code is certainly more complicated with all of the edge cases